src/util/layout.tsx

```diff
@@ -0,0 +1,65 @@
+import { DependencyList, useLayoutEffect, useRef } from "react";
+import {
+  getElementAbsoluteHeight,
+  getHeightOffsetStyle,
+  getNegativeMargin,
+} from "./helpers";
+import useEventListener from "@use-it/event-listener";
+
+type adjustAppLayoutHeightArgs = {
+  dependencies?: DependencyList;
+};
+
+export const useAdjustAppLayoutHeight = ({
```

Comment:
This workaround shouldn't be needed. When I comment out the body of this it still aligns nicely on all pages. Only pages with forms have trouble, but that is because the `updateMaxHeight` function the forms use to calculate their height, doesn't consider the new status bar. I see you added some adjustments for the status bar to the `ScrollableTable` component, which might be needed for the `updateMaxHeight` as well.
After that we should be able to remove this workaround.

Reply:
thank you for suggesting this, I've made necessary changes in updateMaxHeight and it now works on form based pages :+1: